packages/frontend/src/app/modules/common/components/configs-form/connector-action-or-config.ts

```diff
@@ -25,6 +26,10 @@ export interface PieceConfig {
 	required: boolean;
 	extra?: Record<string, unknown>;
 	refreshers?: string[];
+	basicAuthConfigs?: {
+		password: Pick<PieceProperty, "required" | "displayName" | "description">,
```

Comment:
Please remove required, I did remove it in this pr because it has always to be true.